### ISSUE TITLE
Is there a way to work with a video file instead of DVD?

### ISSUE THREAD

**Issue by bjhiltbrand** [rdp/sensible-cinema#25]
Hi, I was looking at using this player and am really interested in it. However, most of my media is in the form of mp4 and avi files. I have looked all over, but I don't see anywhere where I can select a video file for creating an EDL file. Am I missing something? If it's not available as functionality, can it be added? Thanks. :)

**Comment by misbach** [comment]
@bjhiltbrand I know, right! It's surprising that this functionality doesn't exist.

**Comment by rdp** [comment]
It should be there already?

**Comment by bjhiltbrand** [comment]
I found it! For @misbach, if you open the advanced sensible cinema, the option is the one highlighted in the screenshot I attached. Once you click that, you just choose the "file" option and select the file.

@rdp - Well done on this. I really like the parser. It's awesome! I will start looking into the source code and see how I can contribute. Thanks.

![sensible cinema](https://user-images.githubusercontent.com/11344821/42014179-1d032dcc-7a5e-11e8-9f4d-25b1f8942bdf.PNG)
